pages/api/folder/[id].ts

```diff
@@ -0,0 +1,19 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { END_POINT } from "@/lib/constents";
+
+async function hanlder(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
```suggestion
async function handler(req: NextApiRequest, res: NextApiResponse) {
```
함수 이름 오타 인것 같아요.

한가지 덧붙이면, 함수 이름만 보고 어떤 동작을 하는지 나타내는게 중요한데, handler 라는 추상적인 이름은 어떤 동작을 하는지 유추하기 어려울것같아요.

이름 짓는 방법에 관련된 [블로그](https://dkje.github.io/2020/08/03/CleanCodeSeries2-copy/) 글이 있어서 가져왔습니다 참고해보시면 좋을것같아요 :)